flipperdevices/flipperzero-firmware
# Improved Snake with Notification and Score

# What's new
Updated snake to have notifications on capture and a score on the top right.
Inspiration from @whoamins to add code for the score shown and @MuddledBox for the notification on capture

Code for previous Snake PR was reverted to initial state without the unwanted contribution.

# Verification 
- Open snake and play to capture
- See score go up on capture

# Checklist (For Reviewer)
